.gitignore

```diff
@@ -36,3 +36,5 @@ parquet_filtered/
 parquet_final/
 session-manager-plugin.deb
 dictionaries
```

Comment:
What is your rationale to not have
```
*.parquet
```

Reply:
Streamlined the list of file and dir names